asyncpgsa/connection.py

```diff
@@ -15,32 +15,41 @@
 _dialect.paramstyle = 'named'
 
 
+def replace_keys(keys, querystring, params, inline=False):
+    new_query = querystring
+    new_params = []
+    for key, value in enumerate(keys):
+        if inline:
+            new_query = new_query.replace(':' + value, params[key][1], 1)
```

Comment:
Not so, since it only does one replacement at a time, and the replacements are ordered from the initial regex find. Being I know the order, and replace them one at a time in that order, I can conclude that it will only do the replacements I ask. Left-to-Right. I'll write up some tests that put that theory to task.